From eb28f166bb5db8b4a9e9a9bc0fb5a879c1b70d0f Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 2 Jan 2018 15:26:57 +0100
Subject: [PATCH v2 1/1] MAINTAINERS: correct entry for lib/efi*/

lib/efi* indicates files efi* in directory lib.
lib/efi*/ indicates all files in directories lib/efi*.

Fixes: 623b3a579765 efi_selftest: provide an EFI selftest application
Cc: Alexander Graf <agraf@suse.de>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
v2
	rebase on http://git.denx.de/u-boot.git
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index e950267494..9473c00ef6 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -287,7 +287,7 @@ M:	Alexander Graf <agraf@suse.de>
 S:	Maintained
 T:	git git://github.com/agraf/u-boot.git
 F:	include/efi*
-F:	lib/efi*
+F:	lib/efi*/
 F:	test/py/tests/test_efi*
 F:	cmd/bootefi.c
 
-- 
2.14.2

